learning_resources_search/models.py

```diff
@@ -37,6 +43,8 @@ def source_label(self):
 
     def source_description(self):
         channel = self.source_channel()
+        if self.display_label:
+            return self.display_label
         if channel:
             return channel.title
         return self.original_url_params()
```

Comment:
Q: Should something else be displayed that's more user friendly than this when display_label and channel are both None?

![Screenshot 2024-09-26 at 11 07 19 AM](https://github.com/user-attachments/assets/79144730-a195-4b48-b9a0-695d86d64a9e)

Reply:
One thing I found a bit counterintuitive but is outside the scope of this PR: the `source_description` is used as the title in this list while `source_label` is shown underneath that.  I would expect the opposite.

Reply:
working on another pr for sending the emails I can see about changing this there.